lib/semantics/mod-file.cc

```diff
@@ -36,7 +36,7 @@ using namespace parser::literals;
 // The extension used for module files.
 static constexpr auto extension{".mod"};
 // The initial characters of a file that identify it as a .mod file.
-static constexpr auto magic{"!mod$"};
+static const auto magic{"!mod$ v1 sum:"s};
```

Comment:
I think this might now be an instance of a static variable with a runtime constructor, and LLVM forbids their use.

Reply:
OK, I've changed it back to a C string.